package.json

```diff
@@ -46,14 +46,12 @@
 		"@nextcloud/logger": "^2.0.0",
 		"@nextcloud/router": "^2.0.0",
 		"calendar-js": "git+https://github.com/nextcloud/calendar-js.git",
```

Comment:
This is making CI fail

```
npm ERR! Error while executing:
npm ERR! /usr/bin/git ls-remote -h -t ssh://git@github.com/nextcloud/calendar-js.git
npm ERR! 
npm ERR! Warning: Permanently added the RSA host key for IP address '140.82.113.3' to the list of known hosts.
npm ERR! git@github.com: Permission denied (publickey).
npm ERR! fatal: Could not read from remote repository.
```

Maybe move to HTTP ?

Reply:
https://github.com/npm/cli/issues/2610

Reply:
Hacky wrokaround: https://github.com/npm/cli/issues/2610#issuecomment-851347583

Reply:
Another option would be `"calendar-js": "github:nextcloud/calendar-js",` I think.

Reply:
Or use the npm package? See https://github.com/nextcloud/calendar/pull/3371

Reply:
Done! :)